src/allDrugs.jl

```diff
@@ -273,3 +273,46 @@ function blissCellNum(g1s, g2s; T = 96, n = 8)
     @assert(all(combined .>= 0.0))
     return combined
 end
+
+""" temporal combination of two drugs.
+Here we want to plot a simulation, with first 48 hours being treated with lapatinib, and another 48 hours with gem. """
+function temporal_combination(params1, params2, g0, title)
+    t1 = LinRange(0.0, 48.0, 96)
+    t2 = LinRange(48.5, 95.5, 96)
+
+    g1L, g2L = predict(params1, g0, t1, Int(floor(params1[6])), Int(floor(params1[7])), Int(floor(params1[8])), Int(floor(params1[9])))
+    g1G = g2G = zeros(96)
+    nG1 = Int(floor(params2[6]))
+    nG2 = Int(floor(params2[7]))
+    nD1 = Int(floor(params2[8]))
+    nD2 = Int(floor(params2[9]))
+    v = predict2(params2, g1L[end] + g2L[end], t2, nG1, nG2, nD1, nD2,2)
+    g1G[1] = sum(v[1:nG1]) + sum(v[(nG1 + nG2 + 1):(nG1 + nG2 + nD1)])
+    g2G[1] = sum(v[(nG1 + 1):(nG1 + nG2)]) + sum(v[(nG1 + nG2 + nD1 + 1):(nG1 + nG2 + nD1 + nD2)])
+    for j=2:96
+        v = predict2(params2, g1G[j-1]+g2G[j-1], t2, nG1, nG2, nD1, nD2,j+1)
+        g1G[j] = sum(v[1:nG1]) + sum(v[(nG1 + nG2 + 1):(nG1 + nG2 + nD1)])
+        g2G[j] = sum(v[(nG1 + 1):(nG1 + nG2)]) + sum(v[(nG1 + nG2 + nD1 + 1):(nG1 + nG2 + nD1 + nD2)])
+    end
```

Comment:
And here, I'm using the `v` vector that `precidt2` is producing, and calculating each point of G1 and G2 using that vector.
The problem is that it is returning a very large or very small number. I checked it with the first value that we get in `predict`, and `v` was correct. The `g0` I'm providing it every time is very large though.

Reply:
Sorry, I had missed a line.
It works I guess:
![image](https://user-images.githubusercontent.com/34812750/77346500-93c16800-6cf3-11ea-8f35-17fcd9076cbc.png)

Reply:
This is when we use first 48 hours with control, and the next 48 hours with concentration number 7 of gemcitabine
![image](https://user-images.githubusercontent.com/34812750/77347069-822c9000-6cf4-11ea-82f8-da7ffba9c21f.png)